src/modules/vtol_att_control/vtol_type.h

```diff
@@ -210,6 +288,8 @@ class VtolType : public ModuleParams
 	float _ra_hrate_sp = 0.0f;		// rolling average on height rate setpoint for quadchute condition
 
 	hrt_abstime _trans_finished_ts = 0;
+	hrt_abstime _trans_start_ts{0};
+	float _time_since_trans_start{0};
```

Comment:
Not so sure if I like introducing another class variable for that, as normally it can be calculated where used with hrt_elapsed_time(_trans_start_ts) - what was your main motivation for the change, to safe some hrt_ calls?

Reply:
@sfuhrer To me anything that is common for all vtol types belongs into the base class. There is no point why each vtol type should define these variables. I'm actually reducing the number of variables :-)

Furthermore, I need this variable to compute the transition timeout for quadchute which again is something that all vtol types have in common, thus it makes sense to have it here.

Reply:
alright fair, let's keep it.